UnityProject/Assets/Light2D/Scripts/LightingSystem.cs

```diff
@@ -136,7 +136,7 @@ private void Start()
 		{
 			if (GameData.Instance.testServer || GameData.IsHeadlessServer)
 			{
-				Debug.Log("Turn off lightsystem as this is a server");
+				TADB_Debug.Log("Turn off lightsystem as this is a server",TADB_Debug.Category.Light2D.ToString());
```

Comment:
Having to do a `.ToString()` call to use logging categories is absolutely too much boiler plate.